build/scripts/do-release.sh

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh -ex
 
-if [ -z "${CIRCLE_PROJECT_REPONAME}" ] ; then
-  echo "Missing repo name, please set CIRCLE_PROJECT_REPONAME"
-  exit 1
+if [ -z "${GITHUB_REF_NAME}" ] || [ "${GITHUB_REF_TYPE}" != "tag" ] ; then
+    echo "Expected a tag push event, skipping release workflow"
+    exit 1
 fi
 
-if [ -z "${CIRCLE_PULL_REQUEST}" ] && [ -n "${CIRCLE_TAG}" ] && [ "${CIRCLE_PROJECT_USERNAME}" = "weaveworks" ] ; then
-  export RELEASE_DESCRIPTION="${CIRCLE_TAG} (permalink)"
-  RELEASE_NOTES_FILE="docs/release_notes/${CIRCLE_TAG}.md"
+tag="${GITHUB_REF_NAME}"
+
+export RELEASE_DESCRIPTION="${tag} (permalink)"
```

Comment:
same as previous comment - not sure where `RELEASE_DESCRIPTION` is used?

Reply:
It's used in the `.goreleaser.yml` template.